entrypoint.sh

```diff
@@ -23,4 +23,4 @@ cp /etc/sponsorblock/config.json . || cat <<EOF > config.json
     "readOnly": false
 }
 EOF
-node index.js
\ No newline at end of file
+ts-node index.ts
```

Comment:
Is 'ts-node' included in the node:12 docker image?

Reply:
Updated this just to update without thinking :) it most probably isn't even working. This would need an extra step to transpile source to Javascript. There's no need for TS in prod